docs/topics/db/queries.txt

```diff
@@ -970,6 +970,14 @@ field to ``Author``::
     entry.save()
     entry.authors = old_authors # saves new many2many relations
 
+.. note::
```

Comment:
I'm wondering if we should promote this to a `warning`, given the unexpected behavior of not working like `AutoField`.